### ISSUE TITLE
Portal inflammation

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6242]
**Preferred term label:**
Portal inflammation

**Synonyms**
Hepatic portal inflammation

**Definition (free text, please give PubMed ID)**
Infiltration of portal fields by inflammatory cells.

Comment:In nonalcoholic fatty liver disease, portal inflammation is comprised of cells referred to as “chronic”, i.e. lymphocytes, plasma cells, occasional eosinophils, monocytes.
 PMID:19142989 PMID:16939061

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
OMIM:613759 

1/4 patients, PMID:21109225

**Comment by pnrobinson** [comment]
done
